### PR TITLE
code style format: clang-format applied to src/a*/*

### PR DESCRIPTION
**Pull request by keszybz**
[zj: this is a subset of changes generated by clang-format, just the ones
  I think improve readability or consistency.]

This is a part of https://github.com/systemd/systemd/pull/11811.

### CODE REVIEW

File: src/analyze/analyze-security.c [hunk 1]
```diff
@@ -147,11 +152,11 @@ static bool security_info_runs_privileged(const struct security_info *i)  {
...
-                const struct security_assessor *a,
-                const struct security_info *info,
-                const void *data,
-                uint64_t *ret_badness,
-                char **ret_description) {
+        const struct security_assessor *a,
+        const struct security_info *info,
+        const void *data,
+        uint64_t *ret_badness,
+        char **ret_description) {
```
Comment (yuwata) [inline src/analyze/analyze-security.c]: The indentation here should be 16ch.

File: src/analyze/analyze-verify.c [hunk 2]
```diff
@@ -141,9 +141,10 @@ static int verify_executables(Unit *u) {
 
         assert(u);
 
-        exec =  u->type == UNIT_SOCKET ? SOCKET(u)->control_command :
+        exec = u->type == UNIT_SOCKET ?
+                SOCKET(u)->control_command :
                 u->type == UNIT_MOUNT ? MOUNT(u)->control_command :
-                u->type == UNIT_SWAP ? SWAP(u)->control_command : NULL;
+                                        u->type == UNIT_SWAP ? SWAP(u)->control_command : NULL;
```
Comment (yuwata) [inline src/analyze/analyze-verify.c]: Hmm... I do not like this indentation...

File: src/analyze/analyze.c [hunk 3]
```diff
@@ -822,8 +838,13 @@ static int analyze_plot(int argc, char *argv[], void *userdata) {
...
 
-static int list_dependencies_print(const char *name, unsigned level, unsigned branches,
-                                   bool last, struct unit_times *times, struct boot_times *boot) {
+static int list_dependencies_print(
+        const char *name,
+        unsigned level,
+        unsigned branches,
+        bool last,
+        struct unit_times *times,
+        struct boot_times *boot) {
```
Comment (yuwata) [inline src/analyze/analyze.c]: I prefer 16ch indentation here.

File: src/analyze/analyze.c [hunk 4]
```diff
@@ -1582,10 +1601,7 @@ static void dump_syscall_filter(const SyscallFilterSet *set) {
                set->help);
 
         NULSTR_FOREACH(syscall, set->value)
-                printf("    %s%s%s\n",
-                       syscall[0] == '@' ? ansi_underline() : "",
-                       syscall,
-                       ansi_normal());
+                printf("    %s%s%s\n", syscall[0] == '@' ? ansi_underline() : "", syscall, ansi_normal());
```
Comment (yuwata) [inline src/analyze/analyze.c:1605]: Not related to this PR, but we should not use `syscall` for the variable name.

File: src/analyze/analyze-security.c [hunk 5]
```diff
@@ -1916,14 +1921,15 @@ static int acquire_security_info(sd_bus *bus, const char *name, struct security_
...
-                                   info);
+        r = bus_map_all_properties(
+                bus,
+                "org.freedesktop.systemd1",
+                path,
+                security_map,
+                BUS_MAP_STRDUP | BUS_MAP_BOOLEAN_AS_BOOL,
+                &error,
+                NULL,
+                info);
```
Comment (yuwata) [inline src/analyze/analyze-security.c]: I've not followed the previous discussion, but we usually use 16ch indentation for wrapping arguments on calling functions. Though, I do not oppose this change...

File: src/analyze/analyze-verify.c (hunk 2 above)
Comment (keszybz) [reply src/analyze/analyze-verify.c]: Matching `?` and `:` are aligned. I think it's a reasonable change.

**Comment by keszybz** [comment]
You're right about the 16 ch indentation. `git grep -A10 '\($' src/` shows this clearly, and indeed, without that, argument lists would blend in too much with variable lists, which have 8 ch indentation. I removed any indentation changes for arguments to 8 ch that were in the patch.

File: src/analyze/analyze-verify.c (hunk 2 above)
Comment (yuwata) [reply src/analyze/analyze-verify.c]: ??? Sorry, I do not understand... Which `?` and `:` are aligned?

Comment (keszybz) [reply src/analyze/analyze-verify.c]: Github is displaying this wrong, and makes it seem like it's unaligned.
The second and third argument to the ternary operator are aligned to the same column.
This is different than what is done in various other places, but it is quite reasonable.
But if you don't like it, I can drop it.

Comment (yuwata) [reply src/analyze/analyze-verify.c]: > Github is displaying this wrong,

Ah, I see.

I am not sure how we should indent expressions with ternary operators. But, at least in this case, I prefer the original indentation.

Comment (keszybz) [reply src/analyze/analyze-verify.c]: Dropped.

**Pull request merged**